### DISCUSSION TITLE
HC_MISC/pcap/

### DISCUSSION THREAD

**Discussion by giscus** [ProbiusOfficial/Hello-CTF#107] [Show and tell]
# HC_MISC/pcap/

开源免费、新手友好的CTF(Capture The Flag,夺旗赛)入门教程

https://hello-ctf.com/HC_MISC/pcap/

**Comment by eerduosi** [comment]
所以不需要配置，直接开始

**Comment by sea1dream** [comment]
偶，好难啊

**Comment by ZGC1314** [comment]
Comparison operator中的==英文缩写应当是eq。

**Comment by goodlunatic** [comment]
CTF-Misc这部分内容也可以参考这篇文章：
https://goodlunatic.github.io/posts/1ad9200/
